tools/identity-resolution/identity-resolution/Helpers/GitHubAadAliasConverter.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Net.Http;
+using Azure.Sdk.Tools.NotificationConfiguration.Models;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Sdk.Tools.NotificationConfiguration.Helpers
+{
+    public class GitHubToAadAliasResolver
```

Comment:
I would expect the class name and the file name to match here (Resolver vs. Converter).